packages/openid4vc/src/openid4vc-holder/OpenId4VciHolderServiceOptions.ts

```diff
@@ -43,44 +37,47 @@ export interface OpenId4VciNotificationMetadata {
  */
 export type OpenId4VciNotificationEvent = 'credential_accepted' | 'credential_failure' | 'credential_deleted'
 
-export type OpenId4VciTokenResponse = Pick<AccessTokenResponse, 'access_token' | 'c_nonce'>
-
 export type OpenId4VciRequestTokenResponse = {
   accessToken: string
   cNonce?: string
-  dpop?: { jwk: Jwk; nonce?: string }
+  dpop?: OpenId4VciDpopRequestOptions
 }
 
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+type UnionToArrayUnion<T> = T extends any ? T[] : never
```

Comment:
which type does not extend `any`? `never`?

Reply:
It maps from `TypeA | TypeB` to `Array<TypeA> | Array<TypeB>`. I don't really know why it works, but it's different that doing `Array<TypeA | TypeB>`

Reply:
Ahh okay. Maybe add a comment with this in there? Was quite confusing haha